README.md

```diff
@@ -1 +1,33 @@
-# HH-btag
\ No newline at end of file
+# HH-btag   
+Interface providing feature computation for CMS RunII HH-btag  NN to the identification of the reconstructed b-jets for the final state of hh->bbtautau
+
+# Installation
+
+1. cmsrel CMSSW_10_2_20
+1. cd CMSSW_10_2_20/src
+1. cmsenv
+1. git clone git@github.com:hh-italian-group/HHbtag.git
+1. scram b
+
+# Testing
+1. hhBtagTest
+Will produce the resulting scores for the input values for two events
+
+# Features
+
+The features used for the training are:
+
+1. jet_pt: pT of each of the jets
+1. jet_eta: eta of each of the jets
+1. rel_jet_M_pt: Relative mass of the b-jet candidate: jet M / jet pT
+1. rel_jet_E_pt Relative energy of the b-jet candidate: jet E / jet pT
+1. jet_htt_deta: Eta between the b-jet and the HTT candidate. The HTT 4-momentum is calculated as the sum of the 4-momentum of the 2 taus with highest DeepTauVsJet Discriminator)
```

Comment:
Here and below:
- do not specify how exactly taus should be selected (reference the baseline)
- visible 4-momentum of the HTT